Gulpfile.js

```diff
@@ -752,16 +765,11 @@ gulp.task('docker-build', done => {
         }
     }
 
-    const packageId = `${packageInfo.name}-${packageInfo.version}.tgz`;
-    const command = `docker build --no-cache --build-arg packageId=${packageId} -q -t testcafe -f docker/Dockerfile .`;
-    const imageId = childProcess.execSync(command, { env: process.env })
-        .toString()
-        .replace(/\n/g, '');
+    const packageId  = `${packageInfo.name}-${packageInfo.version}.tgz`;
+    const tagCommand = PUBLISH_TAGS.map(tag => `-t ${PUBLISH_REPO}:${tag}`).join(' ');
+    const command    = `docker build --no-cache --build-arg packageId=${packageId} -q ${tagCommand} -f docker/Dockerfile .`;
```

Comment:
Please, remove the `-q` switch here and in the `docker-test` [task](https://github.com/DevExpress/testcafe/pull/3111/files#diff-878989239e07228f7903fde98f450ed9R788) and add the `stdio: 'inherit'` option to the `execSync` call below.